code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -13,6 +13,7 @@
 	var/obj/item/storage/pill_bottle/bottle = null
 	var/mode = 1
 	var/condi = FALSE
+	var/list/chosenPillStyle = 1
```

Comment:
stop it, get some help

Reply:
blame @MrDoomBringer

Reply:
blame @Qustinnus for not checking the code i sent him :^P